tests/integration-tests/test_runner.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import datetime
+import logging
+import multiprocessing
+import os
+import sys
+import time
+
+import argparse
+import pytest
+
+from reports_generator import generate_json_report, generate_junitxml_merged_report
+
+logger = logging.getLogger()
+logging.basicConfig(format="%(asctime)s - %(levelname)s - %(module)s - %(message)s", level=logging.INFO)
+
+START_TIME = time.time()
+START_TIME_ISO = datetime.datetime.fromtimestamp(START_TIME).isoformat()
+
+LOGS_DIR = "{0}.logs".format(START_TIME)
+OUT_DIR = "{0}.out".format(START_TIME)
+
+TEST_DEFAULTS = {
+    "parallelism": None,
+    "retry_on_failures": False,
+    "features": "",  # empty string means all
+    "regions": [
+        "us-east-1",
```

Comment:
What about gov-cloud and other partitions?

Reply:
regions will be selected explicitly when running in the other partitions with the --regions option. It doesn't make much sense to have regions belonging to different partitions in the default since in the end the account used for testing can't have cross partition access.